src/axom/inlet/tests/inlet_object.cpp

```diff
@@ -22,14 +22,14 @@ using axom::inlet::InletType;
 using axom::inlet::LuaReader;
 using axom::sidre::DataStore;
 
-std::shared_ptr<Inlet> createBasicInlet(DataStore* ds,
-                                        const std::string& luaString,
-                                        bool enableDocs = true)
+Inlet createBasicInlet(DataStore* ds,
+                       const std::string& luaString,
+                       bool enableDocs = true)
 {
-  auto lr = std::make_shared<LuaReader>();
+  auto lr = axom::inlet::cpp11_compat::make_unique<LuaReader>();
```

Comment:
Now that I think about it, I think the `cpp11_compat` version can be relegated to internal usage only, at least right now, since all the tests/examples use `LuaReader` which requires C++14.  If a C++11-compatible reader (e.g. JSON) is added, perhaps this approach can be revisited.

Reply:
Thanks -- I think that would help.